surfactant/filetypeid/id_magic.py

```diff
@@ -162,6 +162,18 @@ def identify_file_type(filepath: str) -> Optional[str]:
                 int.from_bytes(magic_bytes[0:4], byteorder="big", signed=False) & 0xFF0F80FF
             ) == 0xF00D0000:
                 return "OMF_LIB"
+            # Check for script files
+            if magic_bytes[:12] == b"#!/bin/bash\n" or suffix == ".sh":
+                return "BASH"
+            if suffix == ".py":
+                return "PYTHON"
+            # Check for web related files
+            if suffix == ".js":
+                return "JAVASCRIPT"
+            if suffix == ".css":
+                return "CSS"
+            if magic_bytes[:16] == "<!DOCTYPE html>\n" or suffix == ".html":
```

Comment:
`.htm` is another common extension for HTML files. I think syntax `suffix in (".html", ".htm")` can be used to avoid a long list of `or` conditions.